bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/DropDownImpl.java

```diff
@@ -62,4 +69,16 @@ public Boolean isMultiSelect() {
     public String getFieldType() {
         return super.getFieldType(FieldType.DROP_DOWN);
     }
+
+    @Override
+    public Boolean isAutoComplete() {
+        return autoComplete;
+    }
+
+    @Override
+    public @NotNull Map<String, Object> getProperties() {
+        Map<String, Object> customProperties = super.getProperties();
+        customProperties.put(ReservedProperties.FD_AUTOCOMPLETE, isAutoComplete());
```

Comment:
I don't think we need this property for headless, since there is no use-case, let's remove it for now.